2019/02/20-22:11:12
Running: clear
------------------------------------------
OUPUT:
[3J[H[2J
ERROR:
None
STATUS: 0
2019/02/20-22:11:32
Running: ls -la
------------------------------------------
OUPUT:
total 80
drwxrwxr-x 4 kevindick kevindick 4096 Feb 20 22:11 .
drwxrwxr-x 6 kevindick kevindick 4096 Feb 20 16:06 ..
-rwxrwxr-x 1 kevindick kevindick 2014 Feb 20 22:02 cli-over-db.py
-rw-rw-r-- 1 kevindick kevindick   13 Feb 20 22:11 commands.txt
-rw-rw-r-- 1 kevindick kevindick 4096 Feb 20 22:11 .commands.txt.swp
drwxrwxr-x 7 kevindick kevindick 4096 Feb 20 22:10 .git
-rw-rw-r-- 1 kevindick kevindick 1062 Feb 20 16:06 LICENSE
-rw-rw-r-- 1 kevindick kevindick  154 Feb 20 22:11 output.txt
-rw-rw-r-- 1 kevindick kevindick  862 Feb 20 16:14 README.md
drwxrwxr-x 2 kevindick kevindick 4096 Feb 20 20:15 scripts

ERROR:
None
STATUS: 0
2019/02/20-22:11:52
Running: pwd
------------------------------------------
OUPUT:
/home/kevindick/Dropbox/Biocluster/cloud-cli

ERROR:
None
STATUS: 0
2019/02/20-22:12:16
Running: python ./scripts/hello.py
------------------------------------------
OUPUT:
Hello World
0
1
2
3
4
5
6
7
8
9
Execution Complete!~

ERROR:
None
STATUS: 0
2019/02/20-23:30:39
Running: python ./scripts/hello.py
------------------------------------------
OUPUT:
Hello World
0
1
2
3
4
5
6
7
8
9
Execution Complete!~

ERROR:
None
STATUS: 0
2019/02/20-23:31:12
Running: pkill cli-over-db.py
------------------------------------------
OUPUT:

ERROR:
None
STATUS: 1
2019/02/20-23:32:03
Running: pkill -f "cli-over-db.py"